### PR TITLE
Add Linter to Tests folder

### PR DESCRIPTION
**Pull request by cdr-chakotay**
Add linting of jUnit tests and reconfigure the linting.

Lint throught the jUnit tests

### CODE REVIEW

File: src/test/java/io/tus/java/client/TestTusClient.java [hunk 1]
```diff
@@ -210,6 +256,7 @@ public void testResumeOrCreateUpload() throws IOException, ProtocolException {
         assertEquals(uploader.getUploadURL(), new URL(mockServerURL + "/foo"));
     }
 
+
     @Test
     public void testResumeOrCreateUploadNotFound() throws IOException, ProtocolException {
```
Comment (cdr-chakotay) [inline src/test/java/io/tus/java/client/TestTusClient.java:261]: @Acconut can you tell me what this test is for ? I have an assumption, but I don't want to write here any wrong information

Comment (Acconut) [reply src/test/java/io/tus/java/client/TestTusClient.java:261]: This test ensures that when a 404 is returned for the HEAD request, the uploader does not error out directly but instead creates a new upload. Makes sense? :)

Comment (cdr-chakotay) [reply src/test/java/io/tus/java/client/TestTusClient.java:266]: Yes makes sense to me! Thank you :)

**Comment by cdr-chakotay** [comment]
Maybe it would be a good idea if you check a few samples to make sure I got everything right. I'm not that deep into the TUS project so far.

**Review by Acconut — APPROVED** [review]
Looks all very good! Thank you for the hard work!

**Comment by cdr-chakotay** [comment]
> Looks all very good! Thank you for the hard work!
Thank you :)

**Pull request merged**